### PR TITLE
PLA-1266 vehicle ID transfer

### PR DESCRIPTION
**Pull request by LorranSutter**

### CODE REVIEW

**Comment by elffjs** [comment]
To be fair, you didn't tag anyone.

File: contracts/NFTs/VehicleId.sol [hunk 1]
```diff
@@ -1,9 +1,12 @@
 //SPDX-License-Identifier: BUSL-1.1
 pragma solidity ^0.8.13;
 
-import "./Base/MultiPrivilege/MultiPrivilege.sol";
+import "../interfaces/IDimoRegistry.sol";
+import "./Base/MultiPrivilege/MultiPrivilegeTransferable.sol";
 
 contract VehicleId is Initializable, MultiPrivilege {
+    IDimoRegistry private _dimoRegistry;
```
Comment (elffjs) [inline contracts/NFTs/VehicleId.sol:8]: Are we just adding this in case we need it later?

**Review by elffjs — APPROVED** [review]

File: contracts/NFTs/VehicleId.sol (hunk 1 above)
Comment (LorranSutter) [reply contracts/NFTs/VehicleId.sol:8]: That is the idea, bc the other NFTs also have it ("uniformity"). Maybe it is dumb

Comment (elffjs) [reply contracts/NFTs/VehicleId.sol:8]: Seems fine to me.

**Pull request merged**